### PR TITLE
Added a simple build command based on instructions in README.md and .circleci/config.yml

### PR DESCRIPTION
**Pull request by ashishkf**
### What does this PR do?

A simple build script to build dd-agent

### Motivation

Automate build for dd-agent

### Additional Notes

Works only on MacOS

### Describe your test plan

Successfully built dd-agent

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+python3 -m venv venv3
+inv deps
+source venv3/bin/activate
+pip3 install -r requirements.txt
```
Comment (pankaj-kf) [inline build.sh:5]: Shouldn't some of this stuff belong to setup.sh? What's inv?

Comment (jt-kloudfuse) [reply build.sh:5]: http://docs.pyinvoke.org/en/stable/index.html

**Review by jt-kloudfuse — COMMENTED** [review]
Script is fine.

There is already a circleci config in the repo so this isn't needed

File: build.sh (hunk 1 above)
Comment (ashishkf) [reply build.sh:5]: I can move the creation of venv outside but the activate needs to be done on per shell basis.

**Comment by ashishkf** [comment]
> Script is fine.
> 
> There is already a circleci config in the repo so this isn't needed

The script is only for local builds and mostly a stop gap measure till we write a build script or make ci work locally. I just did not wanted to forget the steps I took to build the dd-agent locally.

**Pull request merged**